ansible/roles/tldraw-valkey/templates/values.yml.j2

```diff
@@ -0,0 +1,59 @@
+storage:
+  className: "{{ SC_DEFAULT_STORAGE_CLASS_NAME }}"
+
+haMode:
+  enabled: true
+  replicas: {{ TLDRAW_VALKEY_REPLICAS }}
+  masterGroupName: myprimary
+
+image:
+  registry: "docker.io"
+  repository: "valkey/valkey"
+  tag: "{{ TLDRAW_VALKEY_IMAGE_VERSION }}"
+
+resources:
+  limits:
+    cpu: "{{ TLDRAW_VALKEY_CPU_LIMITS|default('1000m', true) }}"
+    memory: "{{ TLDRAW_VALKEY_MEMORY_LIMITS|default('4Gi', true) }}"
+  requests:
+    cpu: "{{ TLDRAW_VALKEY_CPU_REQUESTS|default('100m', true) }}"
+    memory: "{{ TLDRAW_VALKEY_MEMORY_REQUESTS|default('1Gi', true) }}"
+
+sentinelResources:
+  limits:
+    cpu: "{{ TLDRAW_VALKEY_CPU_LIMITS|default('1000m', true) }}"
+    memory: "{{ TLDRAW_VALKEY_MEMORY_LIMITS|default('4Gi', true) }}"
+  requests:
+    cpu: "{{ TLDRAW_VALKEY_CPU_REQUESTS|default('100m', true) }}"
+    memory: "{{ TLDRAW_VALKEY_MEMORY_REQUESTS|default('1Gi', true) }}"
```

Comment:
Shoud have the sentinel container the same amount of resources as the Valkey container at the pod?